docs/DOCKER.md

```diff
@@ -65,3 +65,30 @@ $ docker logs lnd-testnet
 
 This is a simple example, it is possible to use any command-line options necessary
 to expose RPC ports, use `btcd` or `bitcoind`, or add additional chains.
+
+## LND Development and Testing
+
+To test the Docker production image locally, run the following from
+the project root:
+
+```
+$ docker build . -t lnd:1.0
```

Comment:
What's the reason for the version `1.0`? Wouldn't it make more sense to call it something like `-t lnd:master`?

Reply:
typically tag is a version number, although I like the idea of using `master` since `master` is the default.  Happy to make that change.